@xen-orchestra/lite/src/components/pool/dashboard/ramUsage/VmsRamUsage.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <UiCardTitle
-    subtitle
+    :subtitleClass="'subtitle-with-underline'"
```

Comment:
Static prop. You don't need prefix it with `:`.
```suggestion
    subtitleClass="subtitle-with-underline"
```